pkg/utils/utils.go

```diff
@@ -20,9 +24,29 @@ func FletcherChecksum8(inp string) uint8 {
 }
 
 func ShortHostname() (shortName string, err error) {
-	hostname, err := os.Hostname()
-	if err != nil {
-		panic(err)
+	var hostname string
+
+	if filePath, ok := os.LookupEnv("RUNTIMECFG_HOSTNAME_PATH"); ok {
+		dat, err := ioutil.ReadFile(filePath)
+		if err != nil {
+			log.WithFields(logrus.Fields{
+				"filePath": filePath,
+			}).Error("Failed to read file")
+			panic(err)
+		}
+		hostname = strings.TrimSuffix(string(dat[:]), "\n")
```

Comment:
why is the `[:]` necessary?
Why not just run TrimSpace?

Reply:
My bad, thanks!